Java/math/tribonacci.java

```diff
@@ -0,0 +1,54 @@
+ package mypack;

```

Comment:
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top